app/src/utils/api.ts

```diff
@@ -165,6 +164,22 @@ export const getDivisionData = async ({
   return res;
 };
 
+export const getCollectionsData = async ({
+  keyword = "",
+  sortID = "date-desc", // TODO: rename sortID TODO: store defaults as constants
+  pageNum = 1,
+  perPage = CARDS_PER_PAGE,
+}: {
+  keyword?: string;
+  sortID?: string;
+  pageNum?: number;
+  perPage?: number; // OPEN QUESTION: perhaps remove the "perPage" field because the API default is 48 and that is what we want.
```

Comment:
Yes.... might be a YAGNI but I would personally err on the side of implementing this in case 48 changes to another number.

Reply:
so keep it?

Reply:
keeping it for now